docs/how-to/enabling-logging.rst

```diff
@@ -0,0 +1,140 @@
+.. meta::
+  :description: Enabling logging in rocFFT
+  :keywords: rocFFT, ROCm, API, documentation, logging
+
+
+.. _enabling-logging:
+
+********************************************************************
+Enabling logging in rocFFT
+********************************************************************
+
+rocFFT can write a variety of log messages to aid troubleshooting. Here are the different
+logs that rocFFT supports.
+
+*  **Trace logging**: Logs the library entry points (for example, ``rocfft_plan_create`` or ``rocfft_execute``) and their parameter
+   values when they are called. Error messages during plan creation and execution are also logged here.
+*  **Bench logging**: Logs the ``rocfft-rider`` command line when a plan is created.
```

Comment:
The name of the log level is "bench" internally but the purpose of the log level is clearer to readers if we spell it out as "benchmark" here.